elements/pfe-tabs/src/pfe-tabs.scss

```diff
@@ -102,8 +121,8 @@ $variables: (
 
 :host(:not([pfe-variant="earth"])) {
   @include browser-query(ie11) {
-    background-color: #{pfe-color(surface--lightest)};
-    color: #{pfe-color(text)};
+    background-color: #{pfe-var(surface--lightest)};
+    color: #{pfe-var(text)};
```

Comment:
(this comment not specific to ie11)
Just confirming an assumption about approach, currently the variants still have more specificity than the root when it comes to vars, for example:

```css
:host([pfe-variant=earth][aria-selected=false]) {
    --pfe-tabs--BackgroundColor: var(--pfe-theme--surface--lighter, #ececec);
}
```
will win over

```css
body {
    --pfe-tabs--BackgroundColor: green;
}
```

![image](https://user-images.githubusercontent.com/456863/83262324-c58afb00-a18a-11ea-8aa4-d8f145fe5aca.png)

Not flagging this as a problem, just confirming this is desired/expected. Even if it were a problem, I'm not sure of an easy way to fix it?

Reply:
I've been pondering this a lot too.  Maybe we should break out on this and brainstorm.  I can put some time on our calendars.